frontend/src/lib-customizations/espoo/employee/assets/i18n/fi.tsx

```diff
@@ -2813,6 +2814,18 @@ export const fi = {
       capacityFactor: 'Lask',
       staffCountRequired: 'Hlökunta'
     },
+    attendanceReservationByChild: {
+      title: 'Lapsikohtaiset läsnäoloajat',
+      description:
+        'Raportti listaa lapsikohtaisesti huoltajien ilmoittamat lähtö- ja tuloajat. Raportti on saatavilla ryhmä ja yksikkökohtaisesti.',
```

Comment:
Typo:  ryhmä ja yksikkökohtaisesti -> ryhmä- ja yksikkökohtaisesti

Reply:
Fixed! Also rebased to include a CVE fix (no conflicts).